website/docs/understanding_ferretdb.md

```diff
@@ -82,8 +82,8 @@ To reference the fourth field in the array, use the dot notation `"animals.3"`.
 
 Here are more examples of dot notations on arrays:
 
-* [Query an array](https://docs.ferretdb.io/Basic_operations/read/#retrieve-documents-containing-a-value-in-an-array)
-* [Update an array](https://docs.ferretdb.io/Basic_operations/update/#update-an-array-element)
+* [Query an array](./basic_operations/read/#retrieve-documents-containing-a-specific-value-in-an-array)
```

Comment:
```suggestion
* [Query an array](./basic_operations/read.md#retrieve-documents-containing-a-specific-value-in-an-array)
```

Reply:
It doesn't work with a `.md` added to it. This is an example: https://docs.ferretdb.io/Basic_operations/read/#retrieve-all-documents-in-a-collection

Reply:
Ok, so what is the difference between that and the previous line? The one above requires `.md` to be absent, and the one below requires `.md` to be present?

Reply:
Ok, so what is the difference between that and the previous line? The one above requires `.md` to be absent, and the one below requires `.md` to be present?

Reply:
@rumyantseva welcome to the club!

Reply:
Yes, that's what I think. But if it strips out the extension when accessing the sections, then it wouldn't matter. Do let me know, thanks.

Absent: https://docs.ferretdb.io/Basic_operations/read/#retrieve-all-documents-in-a-collection
Present: https://docs.ferretdb.io/Basic_operations/read.md/#retrieve-all-documents-in-a-collection